.pre-commit-config.yaml

```diff
@@ -10,7 +10,8 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+    rev: 6.1.0
     hooks:
       - id: flake8
+        args: ['--config=setup.cfg', '--ignore=W503,E741']
```

Comment:
If this works without duplicating the ignores here, i.e. using the `--config=setup.cfg` then it would be better not to duplicate the ignores here. If this doesn't work without them, then it's ok, but still a bit curious as to why the setup.cfg's ignores don't work here.

Reply:
check the links https://github.com/City-of-Helsinki/kukkuu/pull/347#discussion_r1537430431

Reply:
So it's broken? Well, then it's ok I guess.